src/analysis/modals/AnalysisModal.ts

```diff
@@ -504,9 +512,8 @@ export const AnalysisModal = withDisplayName('AnalysisModal')(
           h(
             ButtonPrimary,
             {
-              // TODO: See spinner overlay comment. Change to pendingCreate.status === 'Loading' || errors.
-              disabled: status === 'Loading' || pendingCreate.status === 'Loading' || errors,
-              tooltip: Utils.summarizeErrors(errors),
+              disabled: loading || errors,
+              tooltip: errors ? Utils.summarizeErrors(errors) : loading ? 'Loading' : '',
```

Comment:
Nit: `cond` may be preferable over a nested ternary.

```suggestion
              tooltip: Utils.cond(
                [!!errors, Utils.summarizeErrors(errors)],
                [loading, 'Loading']
              ),
```